Categories/Category/Instance/Groupoids.agda

```diff
@@ -0,0 +1,72 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Instance.Groupoids where
+
+-- The category of groupoids.
+--
+-- This category should maybe be called "Ho(Groupoids)" or "Ho(Gpd)"
+-- instead.  The "homsets" are not the "usual" ones consisting of
+-- functors, but consist instead of equivalence classes of functors up
+-- to natural isomorphism.  This is because homsets here are really
+-- hom-setoids and we pick natural isomorphism as the equivalence
+-- relation for these setoids.
+--
+-- See https://ncatlab.org/nlab/show/Ho%28Cat%29
+
+open import Level
+open import Categories.Category
+open import Categories.Category.Groupoid renaming (Groupoid to IsGroupoid)
+open import Categories.Functor as Fctr using (Functor; _∘F_)
+open import Categories.Functor.Properties using ([_]-resp-Iso)
+import Categories.Morphism.IsoEquiv as IsoEquiv
+open import Categories.NaturalTransformation.NaturalIsomorphism
+  using (NaturalIsomorphism; associator; unitorˡ; unitorʳ; isEquivalence; _ⓘₕ_)
+private
+  variable
+    o ℓ e : Level
+
+-- A groupoid is a category that has a groupoid structure
+--
+-- This is not the same as Categories.Category.Groupoid.Groupoid.
```

Comment:
I agree that the current `Categories.Category.Groupoid.Groupoid` is ill-named and should be `IsGroupoid`. That should be done - and this local definition removed.

I would prefer if things were done in that order though.  But if you're ok with doing both, I'm also ok with pulling this in first and trust that it will shortly be refactored.

Reply:
Done in #55.